src/main/java/core/basesyntax/service/FileWriteService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriteService {
+    void writeToFile(String report,String filePath);
```

Comment:
Do not forget ABSTRACTION
```suggestion
    void writeToFile(String text, String filePath);
```